homeassistant/components/lyric/climate.py

```diff
@@ -179,6 +181,11 @@ def __init__(
         ):
             self._attr_hvac_modes.append(HVACMode.HEAT_COOL)
 
+        # Setup supported fan modes
+        self._attr_fan_modes = device.settings.attributes.get("fan", {}).get(
+            "allowedModes", []
+        )
```

Comment:
Can it be this list is empty? If that is the case may we should only add `ClimateEntityFeature.FAN_MODE` if the list is populated.

Reply:
I added a conditional in `supported_features`. Now `ClimateEntityFeature.FAN_MODE` will only apply if the list is not empty.